docs/tsconfig.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Maybe we should use some known ts config from other repo parts? Or maybe we can see how it looks in `expo/universe/www` and try to keep it consistent with other projects?

Reply:
That's in progress :D I'm working on this on my _"Strict mode branch"_